ApiKeys.js

```diff
@@ -1,11 +1,11 @@
-export default{
+export default {
   firebaseConfig: {
-  apiKey: 'AIzaSyBkS3PCiObUccwpbmryUms6mWJO5LyQ5VY',
-  authDomain: 'easzy-connect.firebaseapp.com',
-  databaseURL: 'https://easzy-connect.firebaseio.com',
-  projectId: 'easzy-connect',
-  storageBucket: 'easzy-connect.appspot.com',
-  messagingSenderId: '381771645172',
-  appId: '1:381771645172:ios:8d49bb6b02f9f38dc19ea1'
-}
+    apiKey: 'AIzaSyBkS3PCiObUccwpbmryUms6mWJO5LyQ5VY',
```

Comment:
ideally, this wouldn't be committed to the public repo...might want to consider making this private or removing this file